kornia/feature/laf.py

```diff
@@ -0,0 +1,398 @@
+# flake8: noqa E127
+# flake8: noqa E128
+
+import kornia
+import math
+import torch
+import torch.nn.functional as F
+import warnings
+try:
+    import matplotlib.pyplot as plt
+    MATPLOTLIB = True
+except:
+    warnings.warn("no matplotlib found, visualize_laf not available ")
+    MATPLOTLIB = False
+
+
+def get_laf_scale(LAF: torch.Tensor) -> torch.Tensor:
+    """
+    Returns a scale of the LAFs
+    Args:
+        LAF: (torch.Tensor): tensor [BxNx2x3] or [BxNx2x2].
+
+
+    Returns:
+        torch.Tensor: tensor  BxNx1x1 .
+
+    Shape:
+        - Input: :math: `(B, N, 2, 3)`
+        - Output: :math: `(B, N, 1, 1)`
+
+    Example:
+        >>> input = torch.ones(1, 5, 2, 3)  # BxNx2x3
+        >>> output = kornia.get_laf_scale(input)  # BxNx1x1
+    """
+    n_dims = len(LAF.size())
+    eps = 1e-10
+    if (n_dims != 4):
+        raise TypeError(
+            "LAF shape should be must be [BxNx2x3]. "
+            "Got {}".format(LAF.size())
+        )
+    out = LAF[..., 0:1, 0:1] * LAF[..., 1:2, 1:2] -\
```

Comment:
in flake8 what i do is to disable the W504 in all the project.

Reply:
Good idea.